agent/consul/state/catalog_test.go

```diff
@@ -5148,7 +5185,6 @@ func setupIngressState(t *testing.T, s *Store) memdb.WatchSet {
 	}
 	assert.NoError(t, s.EnsureConfigEntry(11, wildcardIngress, nil))
 
-	assert.True(t, watchFired(ws))
```

Comment:
Removing these since the behavior is tested elsewhere, and resetting the watches would require additional calls to `s.GatewayServices`.